core/src/Platforms/Android/SystemWebview/AuthenticationActivity.cs

```diff
@@ -119,39 +117,36 @@ protected override void OnResume()
 
             _restarted = true;
 
-            /*_customTabsActivityManager = new CustomTabsActivityManager(this);
-            _customTabsActivityManager.CustomTabsServiceConnected += delegate { _customTabsActivityManager.LaunchUrl(_requestUrl); };*/
             string chromePackageWithCustomTabSupport = GetChromePackageWithCustomTabSupport(ApplicationContext);
 
             if (string.IsNullOrEmpty(chromePackageWithCustomTabSupport))
             {
-                string chromePackage = GetChromePackage();
-                if (string.IsNullOrEmpty(chromePackage))
-                {
-                    const string chromeNotInstalledMessage = " Chrome is not installed on the device, cannot proceed with authentication";
-                    throw CoreExceptionFactory.Instance.GetClientException(
-                        CoreErrorCodes.ChromeNotInstalledError, 
-                        chromeNotInstalledMessage);
-                }
-
-                Intent browserIntent = new Intent(Intent.ActionView);
-                browserIntent.SetData(Uri.Parse(_requestUrl));
-                browserIntent.SetPackage(chromePackage);
+                Intent browserIntent = new Intent(Intent.ActionView, Uri.Parse(_requestUrl));
                 browserIntent.AddCategory(Intent.CategoryBrowsable);
+
+                CoreLoggerBase.Default.Info(
```

Comment:
>Info [](start = 39, length = 4)

Should this be a warning?

Reply:
done.

---
In reply to: [227734902](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1347#discussion_r227734902) [](ancestors = 227734902)